test/TestConfig/src/org/openj9/envInfo/MachineInfo.java

```diff
@@ -0,0 +1,283 @@
+	public String getSpecVersion() {
+		return specVersion;
+	}
+
+	public void setSpecVersion(String specVersion) {
+		this.specVersion = specVersion;
+	}
+
+	public String getJavaVersion() {
+		return javaVersion;
+	}
+
+	public void setJavaVersion(String javaVersion) {
+		this.javaVersion = javaVersion;
+	}
+
+	public String getInstalledMem() {
+		return installedMem;
+	}
+
+	public void setInstalledMem(String installedMem) {
+		this.installedMem = installedMem;
+	}
+
+	public String getFreeMem() {
+		return freeMem;
+	}
+
+	public void setFreeMem(String freeMem) {
+		this.freeMem = freeMem;
+	}
+
+	public String getCpuCores() {
+		return cpuCores;
+	}
+
+	public void setCpuCores(String cpuCores) {
+		this.cpuCores = cpuCores;
+	}
+
+	public String getCpuModel() {
+		return cpuModel;
+	}
+
+	public void setCpuModel(String cpuModel) {
+		this.cpuModel = cpuModel;
+	}
+
+	public String getCpuSpeed() {
+		return cpuSpeed;
+	}
+
+	public void setCpuSpeed(String cpuSpeed) {
+		this.cpuSpeed = cpuSpeed;
+	}
+
+	public String getSysArch() {
+		return sysArch;
+	}
+
+	public void setSysArch(String sysArch) {
+		this.sysArch = sysArch;
+	}
+
+	public String getSysOS() {
+		return sysOS;
+	}
+
+	public void setSysOS(String sysOS) {
+		this.sysOS = sysOS;
+	}
+
+	public String getProcArch() {
+		return procArch;
+	}
+
+	public void setProcArch(String procArch) {
+		this.procArch = procArch;
+	}
+
+	public String getNuma() {
+		return numa;
+	}
+
+	public void setNuma(String numa) {
+		this.numa = numa;
+	}
+
+	public String getSysVirt() {
+		return sysVirt;
+	}
+
+	public void setSysVirt(String sysVirt) {
+		this.sysVirt = sysVirt;
+	}
+
+	public void getRuntimeInfo() {
+		setVmVendor(ManagementFactory.getRuntimeMXBean().getVmVendor());
+		setVmVersion(ManagementFactory.getRuntimeMXBean().getVmVersion());
+		setSpecVendor(ManagementFactory.getRuntimeMXBean().getSpecVendor());
+		setSpecVersion(ManagementFactory.getRuntimeMXBean().getSpecVersion());
+		setJavaVersion(System.getProperty("java.version"));
+
+		setFreeMemory(Runtime.getRuntime().freeMemory());
+		setTotalMemory(Runtime.getRuntime().totalMemory());
+	}
+
+	public void getMachineInfo(String command) {
+		try {
+			Process proc = Runtime.getRuntime().exec(command);
+			BufferedReader sout = new BufferedReader(new InputStreamReader(proc.getInputStream()));
+			String line = sout.readLine();
+			if (command.equals(MachineInfo.UNAME_CMD)) {
+				setUname(line);
+			} else if (command.equals(MachineInfo.SYS_ARCH_CMD)) {
+				setSysArch(line);
+			} else if (command.equals(MachineInfo.PROC_ARCH_CMD)) {
+				setProcArch(line);
+			} else if (command.equals(MachineInfo.SYS_OS_CMD)) {
+				setSysOS(line);
+			} else if (command.equals(MachineInfo.CPU_CORES_CMD)) {
+				setCpuCores(line);
+				if (getCpuCores() == null) {
+					setCpuCores("" + Runtime.getRuntime().availableProcessors());
+				}
+			}
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+
+	public void getSpaceInfo(String path) {
```

Comment:
Probably want to rename this method to printSpaceInfo, or change to behave same as other methods to store to MachineInfo object and print as part of toString()